application/config/autoload.php

```diff
@@ -52,7 +52,7 @@
 |	$autoload['libraries'] = array('database', 'session', 'xmlrpc');
 */
 
-$autoload['libraries'] = array('database', 'session', 'curl', 'OptionsLib', 'Paths', 'Frequency');
+$autoload['libraries'] = array('database', 'session', 'curl', 'OptionsLib', 'Paths', 'Frequency', 'Qra');
```

Comment:
Can u explain the reason for autoload?

Reply:
All in all great work. i like everything which simplifies the codebase.

Reply:
i can load Qra library on each echo_table_col() function
but on this function (echo_table_col()), frequency is on autoload (ex : echo $ci->frequency->hz_to_mhz($row->COL_FREQ); for frequency case). 
so i thinks i better to have the same method; but if team prefer, i can load library on each function

Reply:
there are 4 views, where the function is needed. A Solution can be: load it at every of the 4 ones.
you're absolutely right with frequency - it is also autoloaded (but not nescessary in my opinion).

i'm remarking this, because autoload means it is loaded everywhere at any time (on every page call). Yes it isn't huge, but where should we set the threshold?!

just thinking loud. leave it up to you, as the others already approved it ;)

Reply:
remove autoload and change with a load on each function, no pblm .. tell me and i modify , perhaps it's a better way

Reply:
thank u so much.
perhaps you find a way to eliminate frequencies, too?
Didn't look where it is used, yet.